docs/source/index.rst

```diff
@@ -15,11 +15,22 @@ This project is still very much a work in progress, but we hope that the
 information here will make it easier for newcomers to get started using
 ``Manim``.
 
-.. tip:: 
+.. tip::
 
     All content of the docs is licensed under the MIT license. Especially for the examples
     you encounter: Feel free to use this code in your own projects!
 
+    We are curious to see the awesome projects you build using this library, feel
+    free to share your projects with us `on Twitter <https://twitter.com/manim_community>`_,
+    `Reddit <https://www.reddit.com/r/manim/>`_, or via `Discord <https://discord.gg/mMRrZQW>`_.
+
+    In case you publish your work made with Manim, we would appreciate if you
```

Comment:
Just a small nitpick..
```suggestion
    In case you publish your work made with Manim, we would appreciate it if you
```

Reply:
nah, works fine without the `it` in this case